### PR TITLE
Auto 152 status endpoints

### PR DESCRIPTION
**Pull request by ginnyyadav**
So, this ended up being pretty simple. We are checking to see if we are returned a 200. Otherwise there's nothing else to check. After consulting with lennart there is no standardized response for the status endpoints. 

Note: this will fail on dev1 as currently dev1 is unstable.

### CODE REVIEW

File: tests/API_contract_tests.postman_collection.json [hunk 1]
```diff
@@ -1543,10 +1543,20 @@
...
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "ab2e9641-b4a8-49f4-bf23-0aca771d62d7",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
```
Comment (jh-bate) [inline tests/API_contract_tests.postman_collection.json]: this appears to be redundant?

File: tests/API_contract_tests.postman_collection.json [hunk 2]
```diff
@@ -3593,13 +3603,243 @@
...
+			"item": [
+				{
+					"name": "auth status check",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "dc915f3c-cbe6-454b-8174-fad6045d7e68",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {\r",
```
Comment (jh-bate) [inline tests/API_contract_tests.postman_collection.json]: should we not just use our standardised check here?? 

then we can also assert no token is returned, and at least log the response time 

```
const assert = eval(globals.loadAsserts);
assert.common(200, 300, false, null,'');
```

**Review by jh-bate — CHANGES_REQUESTED** [review]

**Comment by ginnyyadav** [comment]
Sure thing. Wasn't sure if it would be redundant or not since there was nothing else to check but the 200. Will add that in tomorrow.

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**